Lib/sysconfig/__init__.py

```diff
@@ -504,10 +512,10 @@ def _init_config_vars():
     base_prefix = _BASE_PREFIX
     base_exec_prefix = _BASE_EXEC_PREFIX
 
-    try:
-        abiflags = sys.abiflags
-    except AttributeError:
-        abiflags = ''
+    with warnings.catch_warnings():
+        # ignore DeprecationWarning on sys.abiflags change on Windows
```

Comment:
Can you add some `XXX: remove in 3.16` since at that time we won't have a warning anymore?